src/components/RightPane/Map/MapMarker.tsx

```diff
@@ -47,71 +61,75 @@ class MapMarker extends PureComponent<MapMarkerProps> {
                                         top: -0.75rem;
                                         text-align: center; 
                                         color: white" >
-                            ${this.props.index}
+                            ${index}
                         </div>
                     </div>`,
         });
     };
+    let locationString;
+
+    if (acronym) {
+        locationString = (
```

Comment:
Looks like you're assigning this to something that actually isn't a string. Can you rename this? (ideas: `locationLinkElement`, `locationStringElement`)